pkg/cstor/pool/v1alpha2/pool_utils.go

```
@@ -0,0 +1,98 @@
+package v1alpha2
+
+import (
+	api "github.com/openebs/maya/pkg/apis/openebs.io/v1alpha2"
+	zpool "github.com/openebs/maya/pkg/apis/openebs.io/zpool/v1alpha1"
+	blockdevice "github.com/openebs/maya/pkg/blockdevice/v1alpha2"
+	env "github.com/openebs/maya/pkg/env/v1alpha1"
+	zfs "github.com/openebs/maya/pkg/zfs/cmd/v1alpha1"
+	"github.com/pkg/errors"
+	metav1 "k8s.io/apimachinery/pkg/apis/meta/v1"
+)
+
+func getPathForCSPBdevList(bdevs []api.CStorPoolClusterBlockDevice) ([]string, error) {
+	var vdev []string
+	var err error
+
+	for _, b := range bdevs {
+		path, er := getPathForBDev(b.BlockDeviceName)
+		if er != nil {
+			err = ErrorWrapf(err, "Failed to fetch path for bdev {%s} {%s}", b.BlockDeviceName, er.Error())
+			continue
```

lets return error instead of continue'ing in case of errors

---

ok.. this error is handled in the caller